toolbox/githubctl/main.go

```diff
@@ -289,18 +289,20 @@ func DailyReleaseQualification() error {
 	totalRetries := 240 // Max 20 hours wait before we automatically close the PR and fail release qualification.
```

Comment:
Change code to
 	retryDelay := 5 * time.Minute
        maxWaitTime = 20 * time.Hour // Max 20 hours wait before we automatically close the PR and fail release qualification.
 	totalRetries := int( maxWaitTime / retryDelay)

Reply:
Done

Reply:
Done. PTAL